src/checks/pokerus.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  description: 'Illegal pokerus strain for Pokémon',
+  field: 'pokerusStrain',
+  test (pkmn) {
+    const strain = pkmn.pokerusStrain;
+    const duration = pkmn.pokerusDuration;
+
+    if (strain === 0) {
+      return duration === 0;
+    }
+    if (strain >= 0 || strain <= 15) {
+      const maxDays = strain % 4 + 1;
+      return maxDays > duration;
```

Comment:
Shouldn't this be `>=` instead of `>`? I think a legal Pokémon can have duration equal to `maxDays`.

(It might be useful to add a test for this case, too.)